src/styles/blocks/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  box-sizing: border-box;
```

Comment:
move `box-sizing` to `*` selector

Reply:
Explain please:
1.why?
2.In which file i should add this?

Reply:
already received the explanation